src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -137,7 +137,7 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
                         });
                 }
 
-                if (error.body.three_ds_result.result_code === ResultCode.ChallengeShopper) {
+                if (error.body.three_ds_result.code === ResultCode.ChallengeShopper) {
```

Comment:
This breaks the contract for previous versions of the SDK. It should not be an issue because adyen v2 is not live yet, but we need to be mindful when doing these kind of changes.